spinalcordtoolbox/deepseg/models.py

```diff
@@ -131,9 +131,9 @@
     },
     "model_seg_sci_multiclass_sc_lesion_nnunet": {
         "url": [
-            "https://github.com/ivadomed/model_seg_sci/releases/download/r20240130/model-sci-multisite_r20240130.zip"
+            "https://github.com/ivadomed/model_seg_sci/releases/download/r20240729/model_SCIsegV2_r20240729.zip"
```

Comment:
Note:
- for this new release [r20240729](https://github.com/ivadomed/model_seg_sci/releases/tag/r20240729), the nnUNet output structure is kept intact, thus allowing it to be used in the 3DSlicer software as well (as discussed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4531#issuecomment-2181169132).
- while for the previous release [r20240130](https://github.com/ivadomed/model_seg_sci/releases/tag/r20240130), the nested folders were removed

I'm wondering whether the `sct_deepseg` code is already compatible with the nested nnUNet folder structure

Reply:
Aha! You were right! the nested structure is not supported by SCT

<details>
<summary> Error trace </summary>

```python
Removing temporary folders...
Traceback (most recent call last):
  File "/home/GRAMES.POLYMTL.CA/u114716/spinalcordtoolbox/spinalcordtoolbox/scripts/sct_deepseg.py", line 380, in <module>
    main(sys.argv[1:])
  File "/home/GRAMES.POLYMTL.CA/u114716/spinalcordtoolbox/spinalcordtoolbox/scripts/sct_deepseg.py", line 289, in main
    im_lst, target_lst = inference.segment_non_ivadomed(path_model, model_type, input_filenames, thr,
  File "/home/GRAMES.POLYMTL.CA/u114716/spinalcordtoolbox/spinalcordtoolbox/deepseg/inference.py", line 121, in segment_non_ivadomed
    net = create_net(path_model, device)
  File "/home/GRAMES.POLYMTL.CA/u114716/spinalcordtoolbox/spinalcordtoolbox/deepseg/nnunet.py", line 23, in create_nnunet_from_plans
    raise FileNotFoundError(f"No 'fold_*' directories found in model path: {path_model}")
FileNotFoundError: No 'fold_*' directories found in model path: /home/GRAMES.POLYMTL.CA/u114716/spinalcordtoolbox/data/deepseg_models/model_seg_sci_multiclass_sc_lesion_nnunet     
```

</details>

Reply:
updated to handle nested structure here https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4583/commits/4db6ac3c69d15fe58d532634991ec514631f1f6f

Reply:
Great! Thank you!